app/konnect/dev-portal/publish.md

```diff
@@ -3,34 +3,59 @@ title: Publish a Service to the Dev Portal
 no_version: true
 ---
 
-Publish Services to the Dev Portal to expose them to your application
-developers.
-
 Through ServiceHub, you can publish any Service in your catalog and its
-documentation to the Dev Portal.
+documentation to the Developer Portal. Publishing services to the Developer Portal is the only way to expose your service to Developers. Once the Developer Portal is published and available to Developers, they can apply for access by [registering](/konnect/dev-portal/access-and-approval/dev-reg/) a Developer Account. You can also [manage](/konnect/dev-portal/access-and-approval/manage-devs/) access to the Developer Portal from the Konnect Cloud interface.
 
-## Publish a Service
+This doc will cover
 
-1. From the left navigation menu, open the {% konnect_icon servicehub %}
-**ServiceHub** and select a Service.
+* [Publishing a Service](#publish)
+
+* [Unpublishing a Service](#unpublish)
+
+* [Accessing the Developer Portal](#url)
+
+* [Toggling public access to the Developer Portal](#access)
 
-2. Click on the **Actions** dropdown and select **Publish to Portal**.
+## Publish a Service {#publish}
 
-    This publishes all of the Service's version specs to the Dev Portal.
+1. From the left navigation menu, open the **ServiceHub** page and select a
+Service.
 
-3. Access the Dev Portal in any of the following ways:
-    * Click the gear icon for any published Service and select **View in portal**.
-    * From the left navigation menu again, go to **Dev Portal**.
-    From there, click the **Portal URL**.
-    * Directly visit the default Dev Portal URL:
+2. Click on the **Service Actions** dropdown menu and select **Publish to Portal**.
```

Comment:
```suggestion
2. Click on the **Service actions** dropdown menu and select **Publish to portal**.
```

Reply:
We should decide whether it should be "click on the" or "click the" - something to the add to the list.